### PR TITLE
Make the `mongo` shell run with Docker Compose

### LINKED ISSUE #276: Run MongoDB's jstests with dance
We should make a dance runner for MongoDB's jstests.

Before that, we should find a binary for the legacy `mongo` shell or build it ourselves at https://github.com/FerretDB/deps.

DoD: https://github.com/mongodb/mongo/blob/master/jstests/core/basic1.js runs on CI with dance.

* https://github.com/FerretDB/dance/pull/284

### PR DESCRIPTION
**Pull request by b1ron**
Closes #276.

### CODE REVIEW

**Comment by AlekSi** [comment]
Let's move it to https://github.com/FerretDB/deps with proper build steps

**Comment by b1ron** [comment]
> Let's move it to https://github.com/FerretDB/deps with proper build steps

See https://github.com/FerretDB/deps/pull/15/files. Can you let me know if what I have done looks correct as is in the draft.

File: build/deps/mongo.Dockerfile [hunk 1]
```diff
@@ -0,0 +1 @@
+FROM ghcr.io/ferretdb/mongo:6.0.3
```
Comment (AlekSi) [inline build/deps/mongo.Dockerfile:1]:
```suggestion
FROM ghcr.io/ferretdb/legacy-mongo-shell:6.0.3-2
```

See https://github.com/FerretDB/deps/pkgs/container/legacy-mongo-shell

File: Taskfile.yaml [hunk 2]
```diff
@@ -70,6 +70,10 @@ tasks:
     cmds:
       - docker compose exec postgres psql -U postgres -d ferretdb
 
+  mongo:
+    desc: "Run legacy mongo shell"
+    cmds:
+      - docker compose run --rm mongo mongo mongodb://host.docker.internal:27017/ {{.CLI_ARGS}}
```
Comment (AlekSi) [inline Taskfile.yaml:76]:
```suggestion
      - docker compose run --rm mongo mongo mongodb://host.docker.internal:27017/ {{.CLI_ARGS}}

```

**Review by AlekSi — APPROVED** [review]

File: Taskfile.yaml [hunk 3]
```diff
@@ -70,6 +70,11 @@ tasks:
     cmds:
       - docker compose exec postgres psql -U postgres -d ferretdb
 
+  mongo:
+    desc: "Run legacy mongo shell"
```
Comment (rumyantseva) [inline Taskfile.yaml:74]: Should it be the same as in https://github.com/FerretDB/FerretDB/pull/1600/files#diff-cd2d359855d0301ce190f1ec3b4c572ea690c83747f6df61c9340720e3d2425e?

**Review by rumyantseva — COMMENTED** [review]
As far as I can see, we have similar changes in https://github.com/FerretDB/FerretDB/pull/1600, should we follow the same approach in both PRs?

**Comment by AlekSi** [comment]
Merged into #284.

**Pull request closed without merging**